src/main/java/ru/hogwarts/school/controller/InfoController.java

```diff
@@ -0,0 +1,19 @@
+package ru.hogwarts.school.controller;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/info")
+public class InfoController {
+
+    @Value("${server.port}")
+    private int port;
+
+    @GetMapping("/port")
+    public int GetPort() {
```

Comment:
методы именуем с маленькой буквы
```suggestion
    public int getPort() {
```